docs/src/tutorials/tutorial_1.md

```diff
@@ -4,4 +4,336 @@
 
-If you have not installed `Javis` yet, please visit the homepage to [read the installation instructions](../index.html#Installation).
+If you have not installed `Javis` yet, please visit the homepage to [read the installation instructions](/index.html#Installation).
+
+## Explanation
+
+Luxor.jl drawing onto a canvas. It provides simple functions like line, circle and so on and does animation. If you're interested in 2D graphics, you should definitely check out the awesome Luxor Tutorial.
+
+In order to use `Javis.jl`, you will need to include the following in your file.
+
+```julia
+using Javis
+
+javis(
+    [[video]], # Video struct
+    [[actions]], # Vector
+    pathname=[[pathname]] # String
+)
+```
+
+`javis()` has three parameters: 
+* `video`
+* `actions`
+* `pathname`
+
+### pathname
+
+The `pathname` is a String. In order to get the final result of `javis`, you will need to set `pathname` to the path where the gif will be stored and the gif name. Similar to the following:
+
+```julia
+javis(
+    [[video]], # Video struct
+    [[actions]], # Vector
+    pathname="/path/animation.gif"
+)
+```
+
+### Video
+
+The `video` is a Video struct and its constructor arguments are width and height.
+
+```julia
+video = Video(500,500) # 500x500 // width x height
+javis(
+    video,
+    [[actions]], # Vector
+    pathname="/path/animation.gif"
+)
+```
+
+### Actions
+
+The `actions` is a Vector of actions. There are two types of actions:
+
+* `BackgroundAction`
+* `Action`
+
+```julia
+video = Video(500,500)
+javis(
+    video,
+    [
+        ...
+    ],
+    pathname="/path/animation.gif"
+)
+```
+
+#### BackgroundAction()
+
+```julia
+BackgroundAction(
+    [[frames]], # Range (Int:Int)
+    [[func]] # Function call
+)
+```
+
+* `frames`: a range of frames it is applied to.
+* `func`: the function that is called for each frame (in this example, we will be using `ground()`).
+
+##### ground()
+
+```julia
+function ground(args...) 
+    background("white") # canvas background
+    sethue("black") # pen color
+end
+```
+
+The `ground` function sets the background to white and the paint brush color to black.
+
+**Why do I need the `args...`?** Each user function gets three arguments `video, action, frame` in order:
+* `Video`: Video struct
+* `Action`: Action struct
+* `frame`: the current frame number
+
+They are irrelevant for the background such that we don't need to write them down explicitly. "Unfortunately" we need to write `args...` such that Julia actually knows that we have a method that accepts those three arguments. The `...` basically stands for as many arguments as you want.
+
+Note: `sethue` is the same as `setcolor` but doesn't mess with the opacity
+
+
+```julia
+BackgroundAction(
+    1:70, # 70 frames
+    ground # apply ground function for each frame
+)
+```
+
+![](assets/background.gif)
```

Comment:
Either explain that you show the background or leave it out as it's white anyway :wink: